sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Listeners/QueueTargetScaler.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Storage.Queues;
+using Microsoft.Azure.WebJobs.Extensions.Storage.Common.Listeners;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Storage.Common.Listeners
+{
+    /// <summary>
+    /// Provides queue length metrics for the <see cref="QueueTargetScaler"/>.
+    /// </summary>
+    internal sealed class QueueTargetScaler : ITargetScaler
```

Comment:
Should we move the class to `Queue` folder as `QueueScaleMonitor` is there

Reply:
moved this file over